dex/networks/eth/txdata.go

```diff
@@ -28,8 +29,8 @@ const (
 // contract with extra data. It will error if the call data does not call
 // initiate with expected argument types. It returns the array of initiations
 // with which initiate was called.
-func ParseInitiateData(calldata []byte) ([]ETHSwapInitiation, error) {
-	decoded, err := parseCallData(calldata, ETHSwapABI)
+func ParseInitiateData(calldata []byte) ([]swapv0.ETHSwapInitiation, error) {
+	decoded, err := parseCallData(calldata, swapv0.ETHSwapABI)
```

Comment:
I'm likely to work on making ParseInitiateData and other dex/networks/eth helpers version-aware following a chain of other PRs: https://github.com/decred/dcrdex/pull/1307, https://github.com/decred/dcrdex/pull/1313, https://github.com/decred/dcrdex/pull/1320

@buck54321 pls note.  We've chatted about these needing versioning.  While I don't insist on doing it personally, I just wanna coordinate.